src/bundle/Resources/public/js/scripts/admin.search.filters.js

```diff
@@ -1,36 +1,46 @@
 (function(global, doc, eZ, $, flatpickr) {
     let getUsersTimeout;
+    const CLASS_DATE_RANGE = 'ez-filters__range-wrapper';
+    const CLASS_VISIBLE_DATE_RANGE = 'ez-filters__range-wrapper--visible';
     const token = doc.querySelector('meta[name="CSRF-Token"]').content;
     const siteaccess = doc.querySelector('meta[name="SiteAccess"]').content;
     const filterBtn = doc.querySelector('.ez-btn--filter');
     const filters = doc.querySelector('.ez-filters');
     const clearBtn = filters.querySelector('.ez-btn-clear');
     const applyBtn = filters.querySelector('.ez-btn-apply');
-    const selectBtns = doc.querySelectorAll('.ez-btn--select');
-    const dateFields = doc.querySelectorAll('.ez-date-select');
+    const dateFields = doc.querySelectorAll('.ez-filters__range-select');
     const contentTypeSelector = doc.querySelector('.ez-content-type-selector');
     const contentTypeSelect = doc.querySelector('.ez-filters__item--content-type .ez-filters__select');
     const sectionSelect = doc.querySelector('.ez-filters__item--section .ez-filters__select');
     const lastModifiedSelect = doc.querySelector('.ez-filters__item--modified .ez-filters__select');
+    const lastModifiedDateRange = doc.querySelector('.ez-filters__item--modified .ez-filters__range-select');
     const lastCreatedSelect = doc.querySelector('.ez-filters__item--created .ez-filters__select');
+    const lastCreatedDateRange = doc.querySelector('.ez-filters__item--created .ez-filters__range-select');
     const creatorInput = doc.querySelector('.ez-filters__item--creator .ez-filters__input');
     const searchCreatorInput = doc.querySelector('#search_creator');
     const usersList = doc.querySelector('.ez-filters__item--creator .ez-filters__user-list');
     const resetCreatorBtn = doc.querySelector('.ez-filters__item--creator .ez-icon--reset');
     const listGroupsTitle = doc.querySelectorAll('.ez-content-type-selector__group-title');
     const contentTypeCheckboxes = doc.querySelectorAll('.ez-content-type-selector__item [type="checkbox"]');
     const subtreeInput = doc.querySelector('#search_subtree');
+    const dateConfig = {
+        mode: 'range',
+        locale: {
+            rangeSeparator: ' - ',
+        },
+        formatDate: (date) => eZ.helpers.timezone.formatShortDateTime(date, null),
```

Comment:
```suggestion
        formatDate: (date) => eZ.helpers.timezone.formatShortDateTime(date, null, eZ.adminUiConfig.dateFormat.shortDate),
```

To achieve what @katarzynazawada meant we have to add the `dateFormat` to the `formatShortDateTime` helper.